capture-sdk/sdk/src/main/java/net/gini/android/capture/network/AnalysisResult.java

```diff
@@ -31,9 +31,11 @@ public class AnalysisResult extends Result {
      * @param giniApiDocumentId the id of a document in the Gini API
      * @param extractions       the extractions from the Gini API
      */
-    public AnalysisResult(@NonNull final String giniApiDocumentId,
+    public AnalysisResult(
```

Comment:
[Suggestion] Are these breaking changes? As I see, these are public classes which might be used by clients. Please let me know what you think.

Reply:
I'm not sure. We can create extra constructor with some empty text by default. 
Customers which will not use transaction docs will not be affected then